python/mxnet/ndarray/numpy/_op.py

```diff
@@ -732,3 +783,22 @@ def get_list(arrays):
 
     arrays = get_list(arrays)
     return _npi.stack(*arrays, axis=axis, out=out)
+
+
+@set_module('mxnet.ndarray.numpy')
+def slice_assign_scalar(lhs, value, begin, end, step, out=None):  # pylint: disable=too-many-arguments
+    """
+    Assign the rhs to a cropped subset of this ndarray in place. Returns the view of this ndarray.
+    `rhs` and `lhs` should be of the same data type, and on the same device.
+    The shape of rhs should be the same as the cropped shape of this ndarray.
+    """
+    _npi.slice_assign_scalar(lhs, value, begin=begin, end=end, step=step, out=out)
+
+
+@set_module('mxnet.ndarray.numpy')
+def slice_assign(lhs, rhs, begin, end, step, out=None):  # pylint: disable=too-many-arguments
```

Comment:
This not a numpy op and we do not need to define it here.

Reply:
Will resolve it. Thanks!